src/extensions/scratch3_text2speech/index.js

```diff
@@ -648,7 +648,7 @@ class Scratch3Text2SpeechBlocks {
     setVoice (args, util) {
         const state = this._getState(util.target);
 
-        let voice = args.VOICE;
+        let voice = args.VOICE.toUpperCase();
```

Comment:
This breaks when VOICE is not string (and such case is mentioned just below!)

Reply:
Oh, you are right. Maybe use `String(args.VOICE).toUpperCase;`?